addons/knobs/README.md

```diff
@@ -123,8 +123,9 @@ import { text } from '@storybook/addon-knobs';
 const label = 'Your Name';
 const defaultValue = 'Arunoda Susiripala';
 const groupId = 'GROUP-ID1';
+const options = { onChange: ({ name, type, value }) => { handleChange(name, type, value) } }
```

Comment:
Let's either use something like `console.log` here or make it clear that `handleChange` is some user-defined function

Reply:
Done.